src/watchlists/WatchListItem.jsx

```diff
@@ -1,56 +1,83 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Text, View, Image } from 'react-native';
+import {
+  Text, View, Image, TouchableHighlight,
+} from 'react-native';
+import { MaterialIcons } from '@expo/vector-icons';
 import styles from './styles';
 import tradeIcon from '../../assets/icons/watchList/trade.png';
 import detailIcon from '../../assets/icons/watchList/detail.png';
 
-
 export default function watchListItem({
+  isEdit,
```

Comment:
do not change this file at all with this pull request. Keep in mind the Open-closed principle